src/app/src/test/kotlin/org/imperial/mrc/hint/unit/controllers/ADRControllerTests.kt

```diff
@@ -371,7 +417,7 @@ class ADRControllerTests : HintrControllerTests()
     }
 
     @Test
-    fun `pushes file to ADR fails if retrieval from hintr fails`()
+    fun `pushes file to ADR fails retrieval from ADR fails`()
```

Comment:
Please revert this change - the original name is correct because this test appears to relate to retrieval from hintr, not ADR